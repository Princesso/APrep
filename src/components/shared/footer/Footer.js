import React from 'react';
import colors from '../../../styles/colors'
import './footer.css'

const Footer = () => {
  return(
    <div className="footer">
      <div>Copyright made by Oluebube Princess Egbuna Â© 2018</div>
      <div>
        <ul>
          <li><a href="#">Contact</a></li>
          <li><a href="#">FAQs</a></li>
        </ul>
      </div>
    </div>
  )
}

export default Footer