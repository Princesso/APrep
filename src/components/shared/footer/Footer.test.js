import React from 'react';
import Footer from './Footer';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import Store from '../../../store';
import { BrowserRouter, Route, Switch } from 'react-router-dom'

test('Registration Form', () => {
  const component = renderer.create(
    <Provider store={Store}>
      <BrowserRouter>
          <Switch>
            <Footer/>
          </Switch>
      </BrowserRouter>
    </Provider>
  )

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})