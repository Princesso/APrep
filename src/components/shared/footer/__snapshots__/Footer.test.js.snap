// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration Form 1`] = `
<div
  className="footer"
>
  <div>
    Copyright made by Oluebube Princess Egbuna Â© 2018
  </div>
  <div>
    <ul>
      <li>
        <a
          className="link"
          href="/"
          onClick={[Function]}
        >
          Contact
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/"
          onClick={[Function]}
        >
          FAQs
        </a>
      </li>
    </ul>
  </div>
</div>
`;
